src/dotnet/commands/dotnet-help/HelpCommand.cs

```diff
@@ -33,7 +33,8 @@ public class HelpCommand
   run           Compiles and immediately executes a .NET project
   test          Runs unit tests using the test runner specified in the project
   pack          Creates a NuGet package
-  vstest        Runs tests from the specified files";
+  vstest        Runs tests from the specified files
+  add           Group of commands - run `dotnet add --help` for more information";
```

Comment:
My rationale is that this should be less common scenario than anything above so it should be lower but I'm not sure if this should be alphabetic or most common used order